decidim-comments/lib/decidim/comments/test/shared_examples/translatable_comment.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+shared_examples_for "a translated comment event" do
+  describe "translated notifications" do
```

Comment:
This seems like it's a copy and paste with `decidim-meetings/lib/decidim/meetings/test/translated_event.rb` - can we DRY both up? (It applies to the rest of specs)